.travis.yml

```diff
@@ -9,11 +9,14 @@ before_script:
   - pip install numpy
   - pip install opencv-python
   - pip install -r requirements.txt
-  - python origami-lib/test/server_textimageinput.py &
-  - sleep 1
+  - |
+    python origami-lib/test/server_textimageinput.py &
+    SERVER_PID=$!
 
 script:
   - cd origami-lib/test/
   - pytest
+  - echo $SERVER_PID
```

Comment:
echo isn't needed right?

Reply:
It's not necessary but I used it just to show that server pid has a value and the process which is killed is the same.
Should I remove it?

Reply:
Yeah, remove it. It is a good idea to print variables while debugging but we dont need to have this in the production system  :smile: